React/Project structure.md

```diff
@@ -232,6 +232,44 @@ Utility components usually does not have any visual representation on the screen
 When adding `components` folder, you basically extracting smaller chunks of your main component that are not going to be used anywhere else, only in that component.
 
 Note: There _should_ be only one level of `components` folder inside the component folder.
+
+Example:
+
+In this example we have `MainTable` component that has a unique header component that should be places inside `/components` folder because it has some styles, translations and an icon.
```

Comment:
```suggestion
In this example we have the `MainTable` component that has a unique header component that should be placed inside the `/components` folder because it has some styles, translations and an icon.
```